src/server/api/lib/crypto.js

```diff
@@ -0,0 +1,36 @@
+import { getConfig } from "./config";
+const crypto = require("crypto");
+
```

Comment:
Can you document at the top of this file where you got these functions.

I'm not sure there's much beyond the base crypto library that's dependable, but maybe linking to a major company's library that implements the same?

Reply:
This is using the base crypto library. I took the structure from [politics-rewired](https://github.com/politics-rewired/Spoke/blob/master/src/server/api/lib/crypto.js), except I updated it to use the more secure/non-deprecated versions of cipher functions.

The one catch here security-wise is I had to pad the SESSION_SECRET to ensure it's 32 bytes, since the default in .env.example is shorter. But in any production environment that value will be long and randomly generated.

Reply:
Ok, how about you cite:
https://github.com/nodejs/node/blob/master/test/parallel/test-crypto-cipheriv-decipheriv.js
and (while keeping algorithm = "aes-256-cbc"), adapt the code to be similar.

My main point is that I want someone reading this code to see a good source that is likely vetted.  PoliticsRewired doesn't count -- a company that can afford security reviews, basically.